src/borg/remote.py

```diff
@@ -56,6 +56,8 @@ class RepositoryServer:  # pragma: no cover
         'save_key',
         'load_key',
         'break_lock',
+        'get_free_nonce',
+        'commit_nonce_reservation'
```

Comment:
hmm, just an idea:

could this be also stored into the manifest (and loaded / saved with the manifest) instead of adding new api functions to the repo?